src/styles/card.scss

```diff
@@ -0,0 +1,97 @@
+.card {
+  box-sizing: border-box;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  width: 200px;
+  margin: 0 auto;
+  padding-inline: 16px;
+  padding-top: 32px;
+  padding-bottom: 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+  }
```

Comment:
The BEM naming convention suggests using a single underscore for element names, so consider using '.card__image' instead of '.card__image'.